draino/Makefile

```diff
@@ -0,0 +1,13 @@
+RELEASE = "$$(basename $$PWD)"
+
+.PHONY: install
+install:
+	helm upgrade -i --wait $(RELEASE) .
+
+.PHONY: test
+test:
+	echo "no test"
```

Comment:
@taishin 
Can you tell me the reason for the no test?
This is important to know why later.